x-pack/plugins/apm/public/components/app/ErrorGroupDetails/Distribution/index.tsx

```diff
@@ -122,8 +126,21 @@ export function ErrorDistribution({ distribution, title }: Props) {
             ),
             chartStartTime: buckets[0].x0,
             getFormatter,
+            selectedAlertId,
+            setSelectedAlertId,
             theme,
           })}
+          <Suspense fallback={null}>
```

Comment:
Don't we want to show a spinner or something to indicate that the component is loading?

Reply:
It's for the flyout opening from the side so I don't know if a spinner would show on the flyout or somewhere weird. I'll try it and add it in if it looks ok.

Reply:
The JS chunk for the component loads when you go to the overview (it's lazy, so it doesn't load on other pages), so the spinner would never show up. A `Suspense` is required because it's a lazy-loaded component.